internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -4191,6 +4191,51 @@ func TestHandleGetDifficulty(t *testing.T) {
 	}})
 }
 
+func TestHandleGetHeaders(t *testing.T) {
+	t.Parallel()
+
+	hash32 := "349b3e23b64cb4b71d09b9be4652c9e02e73430daee1285ea03d92aa437dcf37"
+	headerBytes, err := block432100.Header.Bytes()
+	if err != nil {
+		panic(err)
```

Comment:
Just a nit, but for consistency these types of errors are being handled with a `t.Fatalf` rather than panicing across the other test functions.